library/src/main/java/com/novoda/downloadmanager/DownloadBatchNotification.java

```diff
@@ -1,58 +0,0 @@
-package com.novoda.downloadmanager;
-
-import android.app.Notification;
-import android.content.Context;
-import android.support.annotation.DrawableRes;
-import android.support.v4.app.NotificationCompat;
-
-class DownloadBatchNotification implements NotificationCreator<DownloadBatchStatus> {
-
-    private static final boolean NOT_INDETERMINATE = false;
-
-    private final Context context;
-    private final int iconDrawable;
-
-    DownloadBatchNotification(Context context, @DrawableRes int iconDrawable) {
-        this.context = context.getApplicationContext();
-        this.iconDrawable = iconDrawable;
-    }
-
-    @Override
-    public NotificationInformation createNotification(String notificationChannelName, DownloadBatchStatus downloadBatchStatus) {
-        DownloadBatchTitle downloadBatchTitle = downloadBatchStatus.getDownloadBatchTitle();
```

Comment:
This code is now in an anonymous class inside `DownloadManagerBuilder` at 🏈

Reply:
Your usage of icons for locating code ❤️